src/app/pages/storage/modules/disks/components/smart-test-result-list/smart-test-result-list.component.ts

```diff
@@ -82,10 +82,11 @@ export class SmartTestResultListComponent implements OnInit {
       },
     }),
     textColumn({
-      title: this.translate.instant('Lifetime'),
-      propertyName: 'lifetime',
+      title: this.translate.instant('Power On Hours Ago'),
+      propertyName: 'poh_ago',
+      toolTip: '"Power On Hours" are how many hours have passed while the disk has been powered on. "Power On Hours Ago" is how many power on hours have passed since each test.',
```

Comment:
Please wrap string with `this.translate.instant` so that it gets extracted into translation messages.